codebuild/spec/buildspec_integ_matrix.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
Should you include this file if we don't use it yet? If it takes a while for us to switch over to the matrix, it could get out of date as there's no mechanism that forces us to update it when we update the integ batch job.

Reply:
removed- will work on it in my fork.